pkg/cmd/taskrun/testdata/TestListTaskRuns-list_taskruns_by_field-selector_negative_case.golden

```diff
@@ -0,0 +1 @@
+Error: failed to list TaskRuns from namespace foo: Error: failed to list TaskRuns from namespace foo: field label not supported: a
```

Comment:
Can you look into why the double `Error:` message is being displayed here in the test output?

Reply:
Fixed it.